lib/supplejack/controllers/helpers.rb

```diff
@@ -279,51 +283,6 @@ def link_to_record(*args, &block)
         link_to(name, url, html_options)
       end
 
-      # Displays the next and/or previous links based on the record and current search
```

Comment:
Removed this as it is an exact duplicate of the function defined above, I confirmed this with diff

Reply:
👍